tokio/src/runtime/task/waker.rs

```diff
@@ -44,12 +44,38 @@ impl<S> ops::Deref for WakerRef<'_, S> {
     }
 }
 
+cfg_trace! {
+    macro_rules! trace {
```

Comment:
perhaps very nitpicky: i might consider giving this a name other than `trace!` so that it's still possible to use the regular `trace!` macro for other logging in this module (which, we don't currently do, but i can imagine potentially wanting to...so maybe we should call this, like, `trace_waker!` or something? idk...

ottoh, we could just deal with that later if it actually becomes necessary, i suppose.

Reply:
So far in Tokio we seem to prefix all tracing macro calls, like `tracing::trace!`. 🤷

Reply:
ah fair enough, nvm!